web/files/files.go

```diff
@@ -1245,8 +1245,14 @@ func FindFilesMango(c echo.Context) error {
 		return err
 	}
 
-	// drop the fields, they can cause issues if not properly manipulated
-	delete(findRequest, "fields")
+	if reqFields, ok := findRequest["fields"].([]interface{}); ok {
+		// Those fields are necessary for the JSON-API response
+		fields := []string{"_id", "_rev", "type", "class"}
+		for _, v := range reqFields {
+			fields = append(fields, v.(string))
+		}
+		findRequest["fields"] = fields
```

Comment:
CouchDB is OK if we have twice the same field?

Reply:
Yes, I tested, it's ok. But I could add a unit test for this